### ISSUE TITLE
Disable qTranslate-XT in a particular value field

### ISSUE THREAD

**Issue by Interna1ta** [qtranslate/qtranslate-xt#652]
Hello,

I am using qTranslate-XT and Woocommerce and I'm trying to disable it only for a particular field.

This particular field is an input with the id="woocommerce_cancelled_order_recipient" where I would like to have the value="tecnics@mail.com, {customer_email}" but I have the value="[:es]tecnics@mail.com, {customer_email}[:en]tecnics@mail.com, {customer_email} [:]"

Plugin versions:
- qTranslate-XT v3.5.1
- Woocommerce v3.5.2

Thanks!

**Comment by herrvigg** [comment]
I can't tell yet if this is a specific problem of integration with WooCommerce or a bug in qT-XT core but i tend to think it's the first. I don't use WC enough to tell but maybe someone else here can help. We have a bunch of issues with WC but i hope we can sort this out better once i have merged the new modules branch.

**Comment by herrvigg** [comment]
Are you using the integration plugin (WC and qT) in its last version 1.4? Consider also switching to this branch where i have integrated it as a module: https://github.com/qtranslate/qtranslate-xt/pull/627.

In general it's easier not have a field not translated than the opposite 🙈 
So the question is what is making it translatable? In any case this should be quite easy to solve. There's also this "Custom Configuration" block in the options where you enforce some JSON so you should be able to disable a field there if it comes as a side-effect from another configuration you can't change.

**Comment by herrvigg** [comment]
To disable you should be able to set the encoding to `none`, something like this:

```
"post":{
	"pages":{"post.php":"", "post-new.php":""},
	"forms":{
		"post":{
			"fields":{
				"woocommerce_cancelled_order_recipient":{"encode":"none"}
			}
		}
	}
}
```

The rest depends on which page / form you have this field.

**Comment by herrvigg** [comment]
@Interna1ta did you try with this `"encode":"none"`? Seen there is no answer i close this for now. Feel free to re-open if it doesn't solve the problem.

**Comment by bichonnages** [comment]
Same thing here. Email addresses should not be translatable on WooCommerce emails settings.
For exemple on https://[website]/wp-admin/admin.php?page=wc-settings&tab=email&section=wc_email_new_order, the email input #woocommerce_new_order_recipient can be translated and if you do enter an email, email cannot be sent anymore. If you leave it blank, its value goes back to the default WP email address it works again since it is not a string with [:xx]email[:xx][:]

**Comment by herrvigg** [comment]
@Interna1ta  @bichonnages i think i found a solution. It was simpler to add more specific jquery selectors instead of trying to come up with a rule to disable. I excluded the email addresses from the translations, translating them is certainly not a good idea.

**Comment by herrvigg** [comment]
Fix released in 3.9.2
